packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -140,29 +140,29 @@ class Notifications {
 }
 
 class Attributes {
-    public isNoIdentityAttribute(attributeId: CoreId | string): ApplicationError {
-        return new ApplicationError("error.runtime.attributes.isNoIdentityAttribute", `Attribute '${attributeId.toString()}' is not an identity attribute.`);
+    public isNotRepositoryAttribute(attributeId: CoreId | string): ApplicationError {
+        return new ApplicationError("error.runtime.attributes.isNotRepositoryAttribute", `Attribute '${attributeId.toString()}' is not a repository attribute.`);
     }
 
-    public identityAttributeHasAlreadyBeenSharedWithPeer(
+    public repositoryAttributeHasAlreadyBeenSharedWithPeer(
         repositoryAttributeId: CoreId | string,
         peer: CoreAddress | string,
         ownSharedIdentityAttributeId: CoreId | string
     ): ApplicationError {
         return new ApplicationError(
-            "error.runtime.attributes.identityAttributeHasAlreadyBeenSharedWithPeer",
-            `Identity attribute '${repositoryAttributeId.toString()}' has already been shared with peer '${peer.toString()}'. Id of shared identity attribute: ${ownSharedIdentityAttributeId.toString()}.`
+            "error.runtime.attributes.repositoryAttributeHasAlreadyBeenSharedWithPeer",
+            `Repository attribute '${repositoryAttributeId.toString()}' has already been shared with peer '${peer.toString()}'. ID of shared identity attribute: ${ownSharedIdentityAttributeId.toString()}.`
```

Comment:
I would understand it in such a way that you can share a RepositoryAttribute in the sense that sharing a RepositoryAttribute results in corresponding own/peer shared copies. 

If you think that writing "ID of shared repository attribute:..." is not good, why is it okay to write "Repository attribute has already been shared with peer" in the sentence before then?

Reply:
I've also just noticed that it makes no sense to talk about the ID of the RepositoryAttribute, but that the ID of the own shared IdentityAttribute must be used instead.
-> So should we use here and below "ID of **own** shared identity attribute:..." (instead of "ID of shared identity attribute:...")?